docs/data/material/components/autocomplete/UseAutocomplete.tsx

```diff
@@ -1,16 +1,15 @@
 import * as React from 'react';
 import { useAutocomplete } from '@mui/base/AutocompleteUnstyled';
-import { styled } from '@mui/material/styles';
-import { autocompleteClasses } from '@mui/material/Autocomplete';
+import { styled } from '@mui/system';
 
 const Label = styled('label')({
   display: 'block',
 });
 
 const Input = styled('input')(({ theme }) => ({
   width: 200,
-  backgroundColor: theme.palette.background.paper,
-  color: theme.palette.getContrastText(theme.palette.background.paper),
+  backgroundColor: theme.palette.mode === 'light' ? '#fff' : '#000',
+  color: theme.palette.mode === 'light' ? '#000' : '#fff',
```

Comment:
I had to test this inside codesandbox as when you the `theme` inside our docs inherits the theme that is injected. IMHO, this is wrong, the `ThemeProvider` should inject `undefined` not the theme to force to rely on the default.